includes/CodingStandards/GoogleSiteKit/Sniffs/Organizational/CommentTagsDescriptionsAlignedSniff.php

```diff
@@ -0,0 +1,247 @@
+<?php
+/**
+ * Descriptions of tags in docblock should be aligned.
+ *
+ * @since n.e.x.t
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace PHP_CodeSniffer\Standards\GoogleSiteKit\Sniffs\Semantic;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+
+/**
+ * Comment Tags Description Aligned Sniff.
+ *
+ * @since n.e.x.t
+ */
+class CommentTagsDescriptionsAligned implements Sniff {
+
+	/**
+	 * Returns the token types that this sniff is interested in.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array(int)
+	 */
+	public function register() {
+		return array( T_DOC_COMMENT_OPEN_TAG );
+	}
+
+	/**
+	 * Processes this sniff, when one of its tokens is encountered.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param \PHP_CodeSniffer\Files\File $phpcs_file The current file being checked.
+	 * @param int                         $stack_ptr  The position of the current token in the
+	 *                                                stack passed in $tokens.
+	 * @return void
+	 */
+	public function process( File $phpcs_file, $stack_ptr ) {
```

Comment:
This sniff should look for the minimum indentation needed, not just that they're aligned. For example, right now this test would (incorrectly) pass for the following:
```php
 * @param array $avar      A description.
 * @param array $bvar      Another description.
```

It should only pass for:
```php
 * @param array $avar A description.
 * @param array $bvar Another description.
```